docs/learn/platform-overview/README.md

```diff
@@ -61,7 +61,7 @@ Avalanche VMs (AVMs) make it easy to define a blockchain-based decentralized app
 ### Creating Your Blockchain and Virtual Machine
 
 Avalanche supports the creation of new instances of the Avalanche VM.
-* [Create a BlockChain Running Subnet-EVM](../../build/tutorials/platform/create-evm-blockchain.md)
+* [Create a BlockChain Running Subnet-EVM](../../build/tutorials/platform/subnet/create-evm-blockchain.md)
 * [Create a BlockChain Running AVM](../../build/tutorials/platform/create-avm-blockchain.md)
```

Comment:
We should also move these pages into subnets
`Create a Virtual Machine`
`Create a Blockchain Running the AVM`
`Avalanche Local Simulator` and this also can be renamed to `Avalanche Subnet Simulator` as it might fit the purpose better? @patrick-ogrady

Reply:
Move the two docs to be under subnet.